x-pack/test/detection_engine_api_integration/security_and_spaces/group9/exception_operators_data_types/ip_array.ts

```diff
@@ -152,7 +152,7 @@ export default ({ getService }: FtrProviderContext) => {
         await waitForSignalsToBePresent(supertest, log, 1, [id]);
         const signalsOpen = await getSignalsById(supertest, log, id);
         const ips = signalsOpen.hits.hits.map((hit) => hit._source?.ip).sort();
-        expect(ips.flat(Number.MAX_SAFE_INTEGER)).to.eql([]);
```

Comment:
this would kill the typescript typecheck by saying the expanded type is infinitely deep